DiscordChatExporter.Gui/ViewModels/RootViewModel.cs

```diff
@@ -107,6 +107,8 @@ protected override async void OnViewLoaded()
                 IsBotToken = _settingsService.LastToken.Type == AuthTokenType.Bot;
                 TokenValue = _settingsService.LastToken.Value;
             }
+            
+            ((App)System.Windows.Application.Current).SetTheme(_settingsService.IsDarkThemeEnabled ? Theme.Dark : Theme.Light);
```

Comment:
I would abstract this away to a static method in our `Theme` class, i.e. `Theme.SetCurrent(...)`.

Reply:
Yeah--I think that's a good idea